buildpack/core/runtime.py

```diff
@@ -7,6 +7,7 @@
 import sqlite3
 import subprocess
 import time
+from distutils.util import strtobool
```

Comment:
`distutils` is deprecated in Python 3.10, could you use the function from [lib.m2ee.util](https://github.com/mendix/cf-mendix-buildpack/blob/eb39e572556ee2802dfa05b8f76226d0bded2fb9/lib/m2ee/util.py#L203-L216)?

Reply:
Thanks @jeohist ! updated as per your suggestion.